### ISSUE TITLE
bug: App crashes while loading images on intel mac

### ISSUE THREAD

**Issue by Smikha** [vietanhdev/anylabeling#31]
Steps to reproduce:
- git clone repo
- setup venv
- install anylabeling and requirements.txt
- run anylabeling

When attempting to import image files, the app crashes with the following error:

```Traceback (most recent call last):
  File "/Users/sm/Documents/repos/anylabeling/.venv/lib/python3.10/site-packages/anylabeling/views/labeling/widgets/canvas.py", line 263, in mouseMoveEvent
    pos = self.transform_pos(ev.localPos())
  File "/Users/sm/Documents/repos/anylabeling/.venv/lib/python3.10/site-packages/anylabeling/views/labeling/widgets/canvas.py", line 899, in transform_pos
    return point / self.scale - self.offset_to_center()
  File "/Users/sm/Documents/repos/anylabeling/.venv/lib/python3.10/site-packages/anylabeling/views/labeling/widgets/canvas.py", line 911, in offset_to_center
    return QtCore.QPoint(x, y)
TypeError: arguments did not match any overloaded call:
  QPoint(): too many arguments
  QPoint(int, int): argument 1 has unexpected type 'float'
  QPoint(QPoint): argument 1 has unexpected type 'float'
```

I've tried to modify the offset_to_center file so that it passes _int_ to the QPoint function, but I still receive the same error.

**Comment by vietanhdev** [comment]
Can you try to install it by single command?
```
pip install anylabeling
```
Tell me if there is still an error. Thank you.

**Comment by vietanhdev** [comment]
Maybe this bug is related to Python 3.10. https://github.com/wkentaro/labelme/issues/1020
Can you try Python 3.8 and 3.9?

**Comment by vietanhdev** [comment]
Fixed this issue on #33. Please pull the latest code or install v0.2.4 from Pypi.
Thank you for your bug report.

**Comment by ryouchinsa** [comment]
Thanks for your great tool AnyLabeling.

[RectLabel](https://rectlabel.com) is an offline image annotation tool for object detection and segmentation.
Although this is not an open source program, you can label polygons using Segment Anything models.

![sam_polygon](https://github.com/ryouchinsa/sam-cpp-macos/assets/1954306/4640e139-c533-4b8c-b27b-e02a401b9bbd)
